<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleBar</class>
 <widget class="QWidget" name="TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonHelp">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonMinimize">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonClose">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
